puffin/src/lib.rs

```diff
@@ -447,15 +504,27 @@ pub struct GlobalProfiler {
 
     next_sink_id: FrameSinkId,
     sinks: std::collections::HashMap<FrameSinkId, FrameSink>,
+    propagate_all_scope_details: bool,
```

Comment:
what is this? docstring please

Is this the same thing that is called `send_all_scopes` and `full_delta` in other places of the codebase?